src/main/java/uk/gov/hmcts/ccd/domain/model/definition/SearchResultDefinition.java

```diff
@@ -28,25 +29,34 @@ public void setFields(SearchResultField[] fields) {
             .collect(Collectors.toList());
     }
 
-    public Map<String, String> getFieldsUserRoles() {
-        Map<String, String> fields = new HashMap<>();
+    public Map<String, List<String>> getFieldsUserRoles() {
+        Map<String, List<String>> fields = new HashMap<>();
```

Comment:
Would it be worth naming this differently so that you don't need to use this.fields to differentiate from the other instance ?

Reply:
Will refactor this now. Thanks for the review